hadoop-ozone/ozone-manager/src/test/java/org/apache/hadoop/ozone/om/TestSnapshotChain.java

```diff
@@ -65,14 +76,15 @@ public void setup() throws Exception {
 
   private SnapshotInfo createSnapshotInfo(UUID snapshotID,
                                           UUID pathPrevID,
-                                          UUID globalPrevID) {
+                                          UUID globalPrevID,
+                                          long creationTime) {
     return new SnapshotInfo.Builder()
         .setSnapshotId(snapshotID)
         .setName("test")
         .setVolumeName("vol1")
         .setBucketName("bucket1")
         .setSnapshotStatus(SnapshotInfo.SnapshotStatus.SNAPSHOT_ACTIVE)
-        .setCreationTime(Time.now())
+        .setCreationTime(creationTime)
```

Comment:
Yeah this is a negative testcase which will fail without the change going in.

Reply:
I doubt it is correct to say "this is a negative testcase" because it is in decreasing order now. May be use random number if you really want it to be a true negative test.

Reply:
Ok yeah that makes sense. Let me do that. I did it for ease.

Reply:
done